android/src/main/kotlin/com/linecorp/flutter_line_sdk/model/AccessToken.kt

```diff
@@ -21,7 +21,7 @@ data class AccessToken(
             val accessToken = loginResult.lineCredential?.accessToken ?: return null
             return AccessToken(
                 accessToken.tokenString,
-                accessToken.expiresInMillis,
+                (accessToken.estimatedExpirationTimeMillis - System.currentTimeMillis()) / 1000,
                 lineIdTokenString
```

Comment:
To algin the [documentation](https://pub.dev/documentation/flutter_line_sdk/latest/flutter_line_sdk/AccessToken/expiresIn.html) and iOS implementation, please return the `expires_in` which we received from server when issuing the token. (The number is fixed and counted from when the server issued the token).

Reply:
Here is [another explanation](https://pub.dev/documentation/flutter_line_sdk/latest/flutter_line_sdk/StoredAccessToken/expiresIn.html) on how to use these `expires_in` values. Basically, we just provide what received from the server as a simple relaying, and leave the framework users to determine how they would want to use or store them.

Reply:
Okay!